ConsumePlugin/ConsumePlugin.fsproj

```diff
@@ -6,88 +6,80 @@
     <OtherFlags>--reflectionfree $(OtherFlags)</OtherFlags>
   </PropertyGroup>
   <ItemGroup>
-    <MyriadSdkGenerator Include="$(MSBuildThisFileDirectory)..\WoofWare.Myriad.Plugins\bin\$(Configuration)\net6.0\WoofWare.Myriad.Plugins.dll"/>
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="myriad.toml"/>
     <Compile Include="AssemblyInfo.fs" />
     <Compile Include="RecordFile.fs"/>
     <Compile Include="GeneratedRecord.fs">
-      <MyriadFile>RecordFile.fs</MyriadFile>
+      <WhippetFile>RecordFile.fs</WhippetFile>
     </Compile>
     <Compile Include="JsonRecord.fs"/>
     <Compile Include="GeneratedJson.fs">
-      <MyriadFile>JsonRecord.fs</MyriadFile>
+      <WhippetFile>JsonRecord.fs</WhippetFile>
     </Compile>
     <Compile Include="PureGymDto.fs"/>
     <Compile Include="GeneratedPureGymDto.fs">
-      <MyriadFile>PureGymDto.fs</MyriadFile>
+      <WhippetFile>PureGymDto.fs</WhippetFile>
     </Compile>
     <Compile Include="RestApiExample.fs"/>
     <Compile Include="GeneratedRestClient.fs">
-      <MyriadFile>RestApiExample.fs</MyriadFile>
+      <WhippetFile>RestApiExample.fs</WhippetFile>
     </Compile>
     <Compile Include="MockExample.fs"/>
     <Compile Include="GeneratedMock.fs">
-      <MyriadFile>MockExample.fs</MyriadFile>
+      <WhippetFile>MockExample.fs</WhippetFile>
     </Compile>
     <Compile Include="MockExampleNoAttributes.fs" />
     <Compile Include="GeneratedMockNoAttributes.fs">
-      <MyriadFile>MockExampleNoAttributes.fs</MyriadFile>
-      <MyriadParams>
-        <IPublicTypeNoAttr>GenerateMock</IPublicTypeNoAttr>
-        <IPublicTypeInternalFalseNoAttr>GenerateMock(false)</IPublicTypeInternalFalseNoAttr>
-        <InternalTypeNoAttr>GenerateMock</InternalTypeNoAttr>
-        <PrivateTypeNoAttr>GenerateMock</PrivateTypeNoAttr>
-        <PrivateTypeInternalFalseNoAttr>GenerateMock(false)</PrivateTypeInternalFalseNoAttr>
-        <VeryPublicTypeNoAttr>GenerateMock</VeryPublicTypeNoAttr>
-        <CurriedNoAttr>GenerateMock</CurriedNoAttr>
-        <TypeWithInterfaceNoAttr>GenerateMock</TypeWithInterfaceNoAttr>
-      </MyriadParams>
+      <WhippetFile>MockExampleNoAttributes.fs</WhippetFile>
```

Comment:
Made the choice to flatten these params, because the contents of this XML node are presented through msbuild only as a string, not as an XML node!